WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -1124,35 +1128,8 @@ public boolean onMenuItemActionExpand(MenuItem item) {
 
             @Override
             public boolean onMenuItemActionCollapse(MenuItem item) {
-                hideSearchMessage();
-                hideSearchSuggestions();
-                hideSearchTabs();
-                resetSearchSuggestions();
-                if (!mIsTopLevel) {
-                    mSettingsMenuItem.setVisible(true);
-                }
-                mCurrentSearchQuery = null;
-
-                if (mBottomNavController != null) {
-                    mBottomNavController.onRequestShowBottomNavigation();
-                }
-
-
-                if (mIsTopLevel) {
-                    if (isFollowingScreen()) {
-                        mSubFilterViewModel.onSubfilterReselected();
-                    } else {
-                        // return to the followed tag that was showing prior to searching
-                        resetPostAdapter(ReaderPostListType.TAG_FOLLOWED);
-                    }
-
-                    mViewModel.onSearchMenuCollapse(true);
-                } else {
-                    // return to the followed tag that was showing prior to searching
-                    resetPostAdapter(ReaderPostListType.TAG_FOLLOWED);
-                }
-
-                return true;
+                requireActivity().finish();
```

Comment:
When the user collapses the SearchView, we want to finish this activity and go back to WPMainActivity + ReaderFragment -> we don't need to do any cleanup as the fragment dies.